images/cdi-cloner/cloner-startup/internal/helpers/size.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2025 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	/*
+		It is a standardized ioctl request code that queries the size of a block device (e.g., disks, partitions) directly from the kernel
+
+		Structure of ioctl Codes :
+
+		Data Type : 0x8008 (upper 16 bits) indicates:
+			Direction : Read from the kernel (0x80).
+			Size : 8 bytes (64 bits), matching the uint64 return type
+		Magic Number : 0x12 (lower 8 bits of the third byte) identifies this as a block device ioctl
+		Command Number : 0x72 (lower 8 bits of the fourth byte) specifies the exact operation (size query)
```

Comment:
```suggestion
		Magic Number : 0x12 (the third byte) identifies this as a block device ioctl
		Command Number : 0x72 (the fourth byte) specifies the exact operation (size query)
```

"lower 8 bits of a byte " is a non-sense =))